modules/customizing-the-login-page.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * web_console/customizing-the-web-console.adoc
+
+[id="customizing-the-login-page_{context}"]
+= Customizing the login page
```

Comment:
since you're documenting this, maybe add the docs for the oauth error page templates, too?